### ISSUE TITLE
1.5.0-beta2 : kops update cluster falsely shows additional iam policies will be created

### ISSUE THREAD

**Issue by dmcnaught** [kubernetes/kops#1782]
Steps to reproduce:
```bash
--- kubernetes/kube-up ‹master› » kops update cluster
Using cluster from kubectl context: <context hidden>

W0203 17:12:09.813261   64236 apply_cluster.go:635] unable to find version information for kops version "1.5.0-beta2" in channel
W0203 17:12:09.813295   64236 apply_cluster.go:702] unable to find version information for kubernetes version "1.5.2" in channel
I0203 17:12:10.509304   64236 executor.go:91] Tasks: 0 done / 75 total; 31 can run
I0203 17:12:11.928127   64236 executor.go:91] Tasks: 31 done / 75 total; 15 can run
I0203 17:12:12.723982   64236 executor.go:91] Tasks: 46 done / 75 total; 25 can run
I0203 17:12:15.488081   64236 executor.go:91] Tasks: 71 done / 75 total; 4 can run
I0203 17:12:15.755706   64236 executor.go:91] Tasks: 75 done / 75 total; 0 can run
Will create resources:
  IAMRolePolicy/additional.nodes.<context hidden>
  	Role                	name:nodes.<context hidden> id:AROAIRLSNRHFQ23MJJMKO

  IAMRolePolicy/additional.masters.<context hidden>
  	Role                	name:masters.<context hidden> id:AROAJ5QSXIE7BQ53LBFEA

Must specify --yes to apply changes
--- kubernetes/kube-up ‹master› » kops update cluster --yes
Using cluster from kubectl context: <context hidden>

W0203 17:12:24.864770   64477 apply_cluster.go:635] unable to find version information for kops version "1.5.0-beta2" in channel
W0203 17:12:24.864805   64477 apply_cluster.go:702] unable to find version information for kubernetes version "1.5.2" in channel
I0203 17:12:26.632130   64477 executor.go:91] Tasks: 0 done / 75 total; 31 can run
I0203 17:12:28.273574   64477 executor.go:91] Tasks: 31 done / 75 total; 15 can run
I0203 17:12:28.963580   64477 executor.go:91] Tasks: 46 done / 75 total; 25 can run
I0203 17:12:32.301731   64477 executor.go:91] Tasks: 71 done / 75 total; 4 can run
I0203 17:12:32.536808   64477 executor.go:91] Tasks: 75 done / 75 total; 0 can run
I0203 17:12:32.536837   64477 dns.go:140] Pre-creating DNS records
I0203 17:12:33.574380   64477 update_cluster.go:204] Exporting kubecfg for cluster
Wrote config for <context hidden> to "/Users/duncan.mcnaught/.kube/config"
Kops has set your kubectl context to <context hidden>

Cluster changes have been applied to the cloud.


Changes may require instances to restart: kops rolling-update cluster

--- kubernetes/kube-up ‹master› » kops rolling-update cluster
Using cluster from kubectl context: <context hidden>

NAME			STATUS	NEEDUPDATE	READY	MIN	MAX	NODES
master-us-east-1a	Ready	0		1	1	1	1
master-us-east-1c	Ready	0		1	1	1	1
master-us-east-1d	Ready	0		1	1	1	1
nodes			Ready	0		1	1	1	1

No rolling-update required
--- kubernetes/kube-up ‹master› » kops update cluster
Using cluster from kubectl context: <context hidden>

W0203 17:13:29.542141   64990 apply_cluster.go:635] unable to find version information for kops version "1.5.0-beta2" in channel
W0203 17:13:29.542177   64990 apply_cluster.go:702] unable to find version information for kubernetes version "1.5.2" in channel
I0203 17:13:30.298392   64990 executor.go:91] Tasks: 0 done / 75 total; 31 can run
I0203 17:13:35.490260   64990 executor.go:91] Tasks: 31 done / 75 total; 15 can run
I0203 17:13:36.172635   64990 executor.go:91] Tasks: 46 done / 75 total; 25 can run
I0203 17:13:39.277528   64990 executor.go:91] Tasks: 71 done / 75 total; 4 can run
I0203 17:13:39.503880   64990 executor.go:91] Tasks: 75 done / 75 total; 0 can run
Will create resources:
  IAMRolePolicy/additional.masters.<context hidden>
  	Role                	name:masters.<context hidden> id:AROAJ5QSXIE7BQ53LBFEA

  IAMRolePolicy/additional.nodes.<context hidden>
  	Role                	name:nodes.<context hidden> id:AROAIRLSNRHFQ23MJJMKO

Must specify --yes to apply changes
```

**Comment by dmcnaught** [comment]
Tested this again with a fresh kops cluster, no mods, and see the same thing.
Im guessing the two IAMRolePolicys are empty, but kops 'thinks' they need creating. When it tries to create, nothing happens.

**Comment by justinsb** [comment]
I'm going to mark this as a P0 to fix in 1.5.1, and put it out as a known-errata in 1.5.0.  Thanks for finding & reporting @dmcnaught

**Comment by geojaz** [comment]
As far as I can tell, this is fixed in 1.5.2. Please reopen if needed.
